src/page-modules/assistant/types.ts

```diff
@@ -11,7 +11,7 @@ export enum DepartureMode {
 export type TripInput = {
   from: GeocoderFeature;
   to: GeocoderFeature;
-  departureMode: DepartureMode;
+  departureMode?: DepartureMode;
```

Comment:
Is it necessary that this is optional? I think it might be more explicit and easy to follow if this is either `departBy` or `arriveBy`. No "hidden knowledge" that `departBy` is default

Reply:
Couple of things with this:

1. Do we want to include the current date when "Now" is set in the URL? Will that affect subsequent searches made?
2. The URL is _very_ long. I thought that by not including the `departureMode` and `departureDate` when "Now" is set could help making it shorter. But we might be at a point where the URL is so long that it doesn't matter.
3. It looked to me that if "Now" is set, the `departureMode` is set to `departBy`. Isn't that "hidden knowledge" the other way?

Reply:
1. I guess we have to decide if we want a linked URL when searched with 'now' should give the same result or not. 
2. I feel you, but I am unsure what is too long. 
3. Good point. The idea was that we shouldn't use the `DepartureDateSelector` component-specific `DepartureDate` type throughout the app. For the rest of the app we really just need to know if it's `departBy` or `arriveBy`. This might be confusing and a better approach is to move the `DepartureDate` type to the assistant page-module.

Reply:
> I guess we have to decide if we want a linked URL when searched with 'now' should give the same result or not

If we go for "Now" on page load we have to know from the URL whether to do that or not. We can omit the `departureDate` and keep the `departureMode` for instance.

- Arrival: `departureMode=arriveBy&departureDate=1698928046`
- Departure: `departureMode=departAt&departureDate=1698928046`
- Now: `departureMode=departBy`

--- 
> The idea was that we shouldn't use the DepartureDateSelector component-specific DepartureDate type throughout the app.

Can use `DepartureMode` in `pages/assistant.tsx` and move the logic that converts from `DepartureMode` to `DepartureDate` to the component (`departureModeToDepartureDate`). Need to add `departureDate` from the query to the props then to get the time.

Reply:
> We can omit the departureDate and keep the departureMode for instance. 

I think this is a good option. We do need `departureMode` when choosing which cursor to use when loading more trip patterns. We could add code to use `nextPageCursor` if `departureMode` is undefined of course, but not sure if that is the best solution. 

> Can use DepartureMode in pages/assistant.tsx and move the logic that converts from DepartureMode to DepartureDate to the component (departureModeToDepartureDate). Need to add departureDate from the query to the props then to get the time.

This sounds good to me!